packages/react-devtools-scheduling-profiler/src/view-base/HorizontalPanAndZoomView.js

```diff
@@ -169,9 +169,16 @@ export class HorizontalPanAndZoomView extends View {
     if (!this._isPanning) {
       return;
     }
+
+    // Don't prevent mouse-move events from bubbling if they are vertical drags.
```

Comment:
Yes. While adding the new feature I noticed that you couldn't click and drag content horizontally because the vertical view stopped interactions from bubbling if it was also dragging. But really it should only do that if you're dragging vertically, and vice versa.